packages/wallet/backend/src/quote/service.ts

```diff
@@ -174,4 +211,76 @@ export class QuoteService implements IQuoteService {
       (Number(params.amount) * conversionRate.rates[params.to]).toFixed()
     )
   }
+
+  public async createExchangeQuote({
+    userId,
+    accountId,
+    assetCode,
+    amount
+  }: CreateExchangeQuote): Promise<QuoteWithFees> {
+    const accountFrom = await Account.query()
```

Comment:
how about moving this complex query into account service and make it more generic so in can be used in both cases
```
async getAccountWithPaymentPointerBy(where: Partial<Account>) {
    const account = await Account.query()
        .where(where)
        .withGraphFetched({ paymentPointers: true })
        .modifyGraph('paymentPointers', (builder) => {
          builder.where({ active: true }).orderBy('createdAt', 'ASC').limit(1)
        }).limit(1)
        .first()

    return account
  }
  ```
  
  and here
  ```
  const accountFrom = await this.deps.accountService.getAccountWithPaymentPointerBy({ id: accountId, userId })
  let accountTo = await this.deps.accountService.getAccountWithPaymentPointerBy({ userId, assetCode })
  ```